### DISCUSSION TITLE
Video Tutorial

### DISCUSSION THREAD

**Discussion by nicolatrentini** [iisgiua/giuaschool#496] [Idee e proposte]
ciao Antonello
ti consiglio di mettere un Video Tutorial sul sito di Giua@School su come installare il programma su Docker , Archivo Zip

**Comment by trinko** [comment]
Il poco tempo libero che ho lo sto dedicando allo sviluppo dell'applicazione.
Se qualcuno volesse fare un videotutorial ne serei contento, ma io proprio non posso rubare altro tempo alla mia vita privata per fare anche questo.
